.storybook/local-preset.js

```diff
@@ -8,3 +11,44 @@ function managerEntries(entry = []) {
 module.exports = {
   managerEntries,
 };
+
+function serve() {
+  // Temporarily create a server as a psuedo server channel. This is throwaway code
+  const app = express();
+
+  app.post("/build", async (req, res) => {
+    let sent = false;
+    res.setHeader("access-control-allow-origin", "*");
+
+    await run({
+      flags: {
+        projectToken: "00baf09dbbe8",
+      },
```

Comment:
@ghengeveld it feels to me like we should pass this as an option, not a flag (we don't need the CLI to do any parsing, etc). However, currently the `getOptions` code will throw if there's no `projectToken` set in the flags. Let's hash this out.